src/scripts.js

```diff
@@ -4,8 +4,8 @@ const friends = document.querySelector('.friends')
 const userStepGoal = document.querySelector('.user-step-goal')
 const communityStepGoal = document.querySelector('.community-step-goal')
 const water = document.querySelector('.water');
-const waterStats = document.querySelector('.water-stats');
 const graph = document.querySelector('.graph');
+const graphTitle = document.querySelector('.title-graph');
```

Comment:
@srslie I added a querySelector for the Graph title to move the Hydration Weekly Stats underneath.  I'm wondering if we will want to change this when we implement the graph but set this up for now.

Reply:
Looks good for now :) I assume a lot of our graph stuff may change with chart.js